ADotNet/Models/Pipelines/GithubPipelines/DotNets/Tasks/Builders/DotNetBuildTaskBuilder.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets.Tasks.Builders
+{
+    public class DotNetBuildTaskBuilder
+    {
+        private DotNetBuildTask dotnetBuildTask;
+
+        public DotNetBuildTaskBuilder()
+        {
+            dotnetBuildTask = new DotNetBuildTask
```

Comment:
Use `this` for private class members references please

https://github.com/hassanhabib/CSharpCodingStandard/blob/master/2.%20Classes%20and%20Interfaces.md#411-referencing